vault `runtime error: index out of range`

I ran this by mistake, but wasn't expecting this...
I also had a second process running iteratively against transit/encrypt at the time, which stopped working... seemed like too much of a coincidence, but unable to reproduce it yet.

```
$ vault write -f
panic: runtime error: index out of range

goroutine 1 [running]:
panic(0x1238ba0, 0xc4200120c0)
	/usr/local/Cellar/go/1.7.5/libexec/src/runtime/panic.go:500 +0x1a1
github.com/hashicorp/vault/command.(*WriteCommand).Run(0xc4201820b0, 0xc42000a1d0, 0x1, 0x1, 0xc420235960)
	/private/tmp/vault-20170207-43377-18mmxus/src/github.com/hashicorp/vault/command/write.go:41 +0x8ec
github.com/hashicorp/vault/vendor/github.com/mitchellh/cli.(*CLI).Run(0xc4200a0180, 0xc4200a0180, 0x27, 0x15bab60)
	/private/tmp/vault-20170207-43377-18mmxus/src/github.com/hashicorp/vault/vendor/github.com/mitchellh/cli/cli.go:154 +0x25a
github.com/hashicorp/vault/cli.RunCustom(0xc42000a1c0, 0x2, 0x2, 0xc4201bff50, 0x0)
	/private/tmp/vault-20170207-43377-18mmxus/src/github.com/hashicorp/vault/cli/main.go:44 +0x4e7
github.com/hashicorp/vault/cli.Run(0xc42000a1c0, 0x2, 0x2, 0xc4200001a0)
	/private/tmp/vault-20170207-43377-18mmxus/src/github.com/hashicorp/vault/cli/main.go:11 +0x56
main.main()
	/private/tmp/vault-20170207-43377-18mmxus/src/github.com/hashicorp/vault/main.go:10 +0x66
```

--- a/command/write.go
+++ b/command/write.go
@@ -32,6 +32,12 @@ func (c *WriteCommand) Run(args []string) int {
 	}
 
 	args = flags.Args()
+	if len(args) < 1 {
+		c.Ui.Error("write requires a path")
+		flags.Usage()
+		return 1
+	}
+
 	if len(args) < 2 && !force {
 		c.Ui.Error("write expects at least two arguments; use -f to perform the write anyways")
 		flags.Usage()

